server/plugin/command.go

```diff
@@ -140,21 +176,30 @@ func azureDevopsUnsubscribeCommand(p *Plugin, c *plugin.Context, commandArgs *mo
 				&model.WebsocketBroadcast{UserId: commandArgs.UserId},
 			)
 
-			return p.sendEphemeralPostForCommand(commandArgs, fmt.Sprintf("Boards subscription with ID: %q is successfully deleted", args[1]))
+			return p.sendEphemeralPostForCommand(commandArgs, fmt.Sprintf("Boards subscription with ID: %q is successfully deleted", subscriptionIDToBeDeleted))
 		}
 	}
 
-	return p.sendEphemeralPostForCommand(commandArgs, fmt.Sprintf("Boards subscription with ID: %q does not exist", args[1]))
+	return p.sendEphemeralPostForCommand(commandArgs, fmt.Sprintf("Boards subscription with ID: %q does not exist", subscriptionIDToBeDeleted))
 }
 
 func azureDevopsListSubscriptionsCommand(p *Plugin, c *plugin.Context, commandArgs *model.CommandArgs, args ...string) (*model.CommandResponse, *model.AppError) {
-	subscriptionList, err := p.Store.GetAllSubscriptions(commandArgs.UserId)
+	// If 4th argument is present then it must be "all-channels"
+	if len(args) >= 4 && args[3] != constants.FilterAllChannels {
```

Comment:
Why use the sign `>=` rather than `==`?

Reply:
there could be chances that by mistake user adds an extra argument at the end of the command so, in that case, instead of invalidating the command we should execute it up to valid params